elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/javadsl/ElasticsearchFlow.scala

```diff
@@ -93,6 +95,55 @@ object ElasticsearchFlow {
       .createWithPassThrough(indexName, typeName, settings, messageWriter)(elasticsearchClient)
       .asJava
 
+  /**
+   * Create a flow to update Elasticsearch with
+   * [[java.util.List[akka.stream.alpakka.elasticsearch.WriteMessage WriteMessage]]s containing type `T`
+   * with `passThrough` of type `C`.
+   * The result status is part of the [[java.util.List[akka.stream.alpakka.elasticsearch.WriteResult WriteResult]]]
+   * and must be checked for successful execution.
+   *
+   * Warning: When settings configure retrying, messages are emitted out-of-order when errors are detected.
+   *
+   * @param objectMapper Jackson object mapper converting type `T` to JSON
+   */
+  def createBulk[T, C](
+      indexName: String,
+      typeName: String,
+      settings: ElasticsearchWriteSettings,
+      elasticsearchClient: RestClient,
+      objectMapper: ObjectMapper
+  ): akka.stream.javadsl.Flow[java.util.List[WriteMessage[T, C]], java.util.List[WriteResult[T, C]], NotUsed] =
+    createBulk(indexName, typeName, settings, elasticsearchClient, new JacksonWriter[T](objectMapper))
+
+  /**
+   * Create a flow to update Elasticsearch with
+   * [[java.util.List[akka.stream.alpakka.elasticsearch.WriteMessage WriteMessage]]s containing type `T`
+   * with `passThrough` of type `C`.
+   * The result status is part of the [[java.util.List[akka.stream.alpakka.elasticsearch.WriteResult WriteResult]]]
+   * and must be checked for successful execution.
+   *
+   * Warning: When settings configure retrying, messages are emitted out-of-order when errors are detected.
+   *
+   * @param messageWriter converts type `T` to a `String` containing valid JSON
+   */
+  def createBulk[T, C](
+      indexName: String,
+      typeName: String,
+      settings: ElasticsearchWriteSettings,
+      elasticsearchClient: RestClient,
+      messageWriter: MessageWriter[T]
+  ): akka.stream.javadsl.Flow[java.util.List[WriteMessage[T, C]], java.util.List[WriteResult[T, C]], NotUsed] =
+    akka.stream.javadsl.Flow
+      .fromFunction[java.util.List[WriteMessage[T, C]], immutable.Seq[WriteMessage[T, C]]] {
+        javaList: java.util.List[WriteMessage[T, C]] =>
+          javaList.asScala.toIndexedSeq
+      }
```

Comment:
Scala 2.11 doesn't do Java lambdas.

Try with 
```
    akka.stream.scaladsl.Flow[java.util.List[WriteMessage[T, C]]]
      .map(_.asScala.toIndexedSeq)
      ...
      .asJava
```

Reply:
Sweet, that fixed it! Thank you!